docs/library/neopixel.rst

```diff
@@ -8,7 +8,7 @@ This module provides a driver for WS2818 / NeoPixel LEDs.
 
 .. note:: This module is only included by default on the ESP8266, ESP32 and RP2
    ports. On STM32 / Pyboard and others, you can either install the
-   ``neopixel`` package using :term:`mip`, or you can download the module
+   ``neopixel`` package using :term:`mip`, or you can `download the module
```

Comment:
I agree it's broken at the moment, but the fix here doesn't render well either, it looks like "or you can download the module directly from and copy it to the filesystem."

I suggest changing it to:
```
...
directly from `micropython-lib
...
```